lib/PhantomEnvironment.js

```diff
@@ -675,12 +698,12 @@ class PhantomEnvironment extends AbstractEnvironment {
       debug('injecting file %s', filePath);
       this._page.injectJs(filePath);
     });
-    return vow.resolve();
+    return Promise.resolve();
   }
 
   injectBrowserEnv() {
     if (this._browserEnvInjected) {
-      return vow.resolve();
+      return Promise.resolve();
     }
 
     debug('.inject()-ing browser env libs');
```

Comment:
Here injects `browser.bundle.js` before it was build via webpack. But now - not, I think this is a browser build of goose-parser. We need to inject it in different way (for example by passing from the point where we define parser and env). Also starting from that `_browserEnvInjected ` meaningless.

Reply:
Agree, it's broken now by moving env into repo (not by this PR). I will create a separate issue for it.

Reply:
Done https://github.com/redco/goose-phantom-environment/issues/12